agent/php_txn.c

```diff
@@ -914,17 +916,17 @@ nr_status_t nr_php_txn_begin(const char* appnames,
     nr_php_txn_log_error_dt_on_tt_off();
   }
 
-#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO
-  if (nr_php_ini_setting_is_set_by_user("opcache.enable")
-      && NR_PHP_PROCESS_GLOBALS(preload_framework_library_detection)) {
-    nr_php_user_instrumentation_from_opcache(TSRMLS_C);
-  }
-#else
-  if (nr_php_ini_setting_is_set_by_user("opcache.preload")
-      && NR_PHP_PROCESS_GLOBALS(preload_framework_library_detection)) {
-    nr_php_user_instrumentation_from_opcache(TSRMLS_C);
+  /*
+   * Only try to instrument preloaded opcache scripts when opcache enabled and
+   * preload is nt null.  If an INI value does not exist, INI_INT returns 0 and
+   * INI_STR returns NULL.
+   */
+  if (NR_PHP_PROCESS_GLOBALS(preload_framework_library_detection)) {
+    if ((1 == INI_INT("opcache.enable"))
```

Comment:
Will this always return a value of "1" if enabled?  Maybe safer to compare that it is not 0 instead?